examples/src/main/scala/org/apache/spark/examples/sql/SparkSQLExample.scala

```diff
@@ -60,7 +60,8 @@ object SparkSQLExample {
 
   private def runBasicDataFrameExample(spark: SparkSession): Unit = {
     // $example on:create_df$
-    val df = spark.read.json("examples/src/main/resources/people.json")
+    val df = spark.read.json(
+      "file:///" + System.getProperty("user.dir") + "/examples/src/main/resources/people.json")
```

Comment:
Yes. This also assumes that user's current directory is $SPARK_HOME. However, without this change, ``` ./bin/run-example sql.SparkSQLExample``` goes failure with default configuration.

Another option is to give a path of people.json/people.text via CLI. I would like to hear your thought.